src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -49,11 +49,9 @@
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "redCard "
```

Comment:
Maybe don't remove the tag and Avatar examples? Would be good to have them

Reply:
avatar example wont work unless the user has same file in the filepath as example. Tag example still there there were too for some reason